aten/src/ATen/core/jit_type.h

```diff
@@ -1543,7 +1524,30 @@ struct CAFFE2_API ClassType : public NamedType {
     return parameterSlots_->at(slot);
   }
 
+  size_t addAttribute(
+      const std::string& name,
+      TypePtr type,
+      bool is_parameter = false);
+
+  void addMethod(Function* method);
+  Function* getMethod(const std::string& name) const;
+
+#ifndef DECOUPLE_JIT
```

Comment:
The name DECOUPLE_JIT is not informative. I also don't see modifications to any build files, so I don't know if it is added correctly. Furthermore, I am surprised we don't already have a build flag indicating we are building for mobile. I am pretty sure C10 already has one.

Reply:
The macro is currently for internal tests. Please refer to the internal diff D18226237 (which this PR is exported from) for testing cases. The reason for a different name is that the other name is for OSS mobile with full JIT. The two names may exist for a while till we fully move to lite interpreter.

Reply:
The name has been changed to "USE_MOBILE_CLASSTYPE".